go.mod

```diff
@@ -1,3 +1,13 @@
 module github.com/bytecodealliance/wasmtime-go
 
-go 1.14
+go 1.18
+
+require github.com/stretchr/testify v1.8.0
```

Comment:
Does this mean that the `wasmtime` package will always depend on this new package? I would expect some sort of "test only" directive here but this looks like it's importing a normal dependency

Reply:
I don't believe that's possible atm.

Reply:
Hm ok in that case though this may not work out because I'd prefer to keep this package slim since it doesn't have any actual dependencies other than Wasmtime itself, but I realize that the tests would still look like they are today which isn't great...

Reply:
Ah @fgsch cleared this up with me in chat. My worries here are unfounded and this should be good to go